bcda/cclf/cclf_test.go

```diff
@@ -115,43 +133,154 @@ func (s *CCLFTestSuite) TestImportCCLF0_SplitFiles() {
 	assert.Equal(cclfFileValidator{totalRecordCount: 6, maxRecordLength: 549}, validator["CCLF8"])
 }
 
+func (s *CCLFTestSuite) TestImportCCLFDirectoryValid() {
```

Comment:
Golang has a feature called table driven testing, where you can run boiler plate code with different inputs and expected outcomes/outputs. We use this in some areas of our codebase, like in the [fileprocessor](https://github.com/CMSgov/bcda-app/blob/master/bcda/cclf/fileprocessor_test.go#L60C4-L60C4).

Table driven testing doesn't necessarily offer any improvements to performance and unless we're testing a handful of different inputs/outputs then it's not terribly useful.

There are some places like `TestImportCCLFDirectoryValid`, `TestImportCCLFDirectoryInvalid`, `TestImportCCLFDirectoryTwoLevels`,  and [this func ](https://github.com/CMSgov/bcda-app/pull/867/files#diff-98db16c8386ea678000e5f338089ea83629113e9e1219ea93bf02ee5d18a781eR196)where it might come in handy, but that's totally up to you!

Reply:
Thank you, Lauren! I'm going to check out table driven testing and see if I can apply it to some of the upcoming test coverage tickets.